content/en/blog/2024/policing-a-sandbox.md

```diff
@@ -148,7 +148,7 @@ package agent_policy
 
 ```
 
-A user is supposed to specify Init-Data to a Confidential Guest in the form of a base64-encoded string in a specific Pod annotation. The Kata Containers runtime will then pass this data on to the Agent in the Guest VM, which will decode the Init-Data and use it to configure the runtime environment of the workload. Crucially, since Init-Data is not trusted at launch we need a way to establish that the policy has not been tampered with in the process.
+A user is supposed to specify Init-Data to a Confidential Guest in the form of a gzipped and base64-encoded string in a specific Pod annotation. The Kata Containers runtime will then pass this data on to the Agent in the Guest VM, which will decode the Init-Data and use it to configure the runtime environment of the workload. Crucially, since Init-Data is not trusted at launch we need a way to establish that the policy has not been tampered with in the process.
```

Comment:
with this the vocabulary accept list change is not needed:
```suggestion
A user is supposed to specify Init-Data to a Confidential Guest in the form of a `gzip` compressed and base64-encoded string in a specific Pod annotation. The Kata Containers runtime will then pass this data on to the Agent in the Guest VM, which will decode the Init-Data and use it to configure the runtime environment of the workload. Crucially, since Init-Data is not trusted at launch we need a way to establish that the policy has not been tampered with in the process.
```

Reply:
Applied